docs/detections/prebuilt-rules/prebuilt-rules-reference.asciidoc

```diff
@@ -300,11 +302,11 @@ and their rule type is `machine_learning`.
 
 |<<cyberark-privileged-access-security-recommended-monitor, CyberArk Privileged Access Security Recommended Monitor>> |Identifies the occurrence of a CyberArk Privileged Access Security (PAS) non-error level audit event which is recommended for monitoring by the vendor. The event.code correlates to the CyberArk Vault Audit Action Code. |[Elastic] [cyberarkpas] [SecOps] [Log Auditing] [Threat Detection] [Privilege Escalation]  |7.14.0 |1
 
-|<<dns-activity-to-the-internet, DNS Activity to the Internet>> |This rule detects when an internal network client sends DNS traffic directly to the Internet. This is atypical behavior for a managed network, and can be indicative of malware, exfiltration, command and control, or, simply, misconfiguration. This DNS activity also impacts your organization's ability to provide enterprise monitoring and logging of DNS, and opens your network to a variety of abuses and malicious communications. |[Elastic] [Network] [Threat Detection] [Command and Control] [Host]  |7.6.0 |10 <<dns-activity-to-the-internet-history, Version history>>
+|<<dns-activity-to-the-internet, DNS Activity to the Internet>> |This rule detects when an internal network client sends DNS traffic directly to the Internet. This is atypical behavior for a managed network and can be indicative of malware, exfiltration, command and control, or simply misconfiguration. This DNS activity also impacts your organization's ability to provide enterprise monitoring and logging of DNS and it opens your network to a variety of abuses and malicious communications. |[Elastic] [Network] [Threat Detection] [Command and Control] [Host]  |7.6.0 |11 <<dns-activity-to-the-internet-history, Version history>>
```

Comment:
```suggestion
|<<dns-activity-to-the-internet, DNS Activity to the Internet>> |This rule detects when an internal network client sends DNS traffic directly to the Internet. This is atypical behavior for a managed network and can be indicative of malware, exfiltration, command and control, or simply misconfiguration. This DNS activity also impacts your organization's ability to provide enterprise monitoring and logging of DNS, and it opens your network to a variety of abuses and malicious communications. |[Elastic] [Network] [Threat Detection] [Command and Control] [Host]  |7.6.0 |11 <<dns-activity-to-the-internet-history, Version history>>
```

Reply:
Done in https://github.com/elastic/detection-rules/pull/1398